Added NorEval, a novel Norwegian benchmark

vmkhlv opened the pull request:
Hi,

I am adding NorEval, a novel benchmark for evaluating Norwegian language understanding and generation in both official written standards: Bokmål and Nynorsk (the minority variant). NorEval consists of 19 existing peer-reviewed datasets and five datasets created from scratch specifically for this benchmark; all datasets are human-created or human-translated & localized. 

We describe NorEval in our README with a reference to our pre-print (according to the template), which provides further details regarding the existing and newly created datasets.

**Comments**
In contrast to the datasets and benchmarks supported in LM Evaluation Harness, NorEval is designed with a focus on multi-prompt evaluation. Each task includes from four to six prompt templates written by Norwegian native speakers. This design enables more robust evaluation by accounting for prompt sensitivity. To support this, we have structured NorEval using both `tag` and `task` fields. This allows to run the evaluation in:

* Single-prompt scenario, using specific tasks (e.g., `noropenbookqa_nob_p1`)
* Multi-prompt scenario, using tags (e.g., `noropenbookqa_nob`)

Please let me know if this logics aligns with the intended use of the framework.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2919) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/EleutherAI/lm-evaluation-harness?pullRequest=2919) before we can accept your contribution.<br/><hr/>**vmkhlv** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/EleutherAI/lm-evaluation-harness?pullRequest=2919) it.</sub>

baberabb commented:
Hi! Thanks for the PR! Could you sign the CLA and also run the pre-commit:
```bash
pip install pre-commit
pre-commit install
pre-commit run --all-files
```

vmkhlv commented:
Hi @baberabb 

following up on your suggestions, I have also refactored tasks organized as several folders: removed duplicate scripts and simplified configs with an overlapping content.

baberabb commented:
LGTM! Thanks for the PR! If you run into any issues in implementing your benchmark or have ideas for improvements, feedback is always welcome!
